client/app/services/usage-graphs/usage-graphs.html

```diff
@@ -0,0 +1,76 @@
+  <h2 translate>Utilization</h2>
+  <hr>
+  <div class="row">
+    <div class="col-md-4 text-center">
+                <div>
```

Comment:
a div that's necessary (its got no classes... 🤔 )?  Less is more when possible, this looks to happen in a few places... maybe the question is, can we get rid of the classless div?